.vsts-ci.yml

```diff
@@ -24,24 +24,27 @@ jobs:
   strategy:
     maxParallel: 6
     matrix:
-      regular:
-        TASK: regular
-      sdist:
-        TASK: sdist
-        PYTHON_VERSION: 3.5
-      bdist:
-        TASK: bdist
-        PYTHON_VERSION: 3.6
-      inference:
-        TASK: if-else
-      mpi_source:
-        TASK: mpi
-        METHOD: source
-        PYTHON_VERSION: 2.7
-      gpu_source:
-        TASK: gpu
-        METHOD: source
-        PYTHON_VERSION: 3.6
+      # regular:
+      #   TASK: regular
+      # sdist:
+      #   TASK: sdist
+      #   PYTHON_VERSION: 3.5
+      # bdist:
+      #   TASK: bdist
+      #   PYTHON_VERSION: 3.6
+      # inference:
+      #   TASK: if-else
+      # mpi_source:
+      #   TASK: mpi
+      #   METHOD: source
+      #   PYTHON_VERSION: 2.7
+      # gpu_source:
+      #   TASK: gpu
+      #   METHOD: source
+      #   PYTHON_VERSION: 3.6
```

Comment:
I am testing an Azure-specific thing, trying to shorten the feedback cycle. Will be sure to uncomment these once I have it working.